examples/kotlin-diktat/fix/smoke/src/main/kotlin/org/cqfn/save/IgnoreLinesTest/IgnoreLines/save.toml

```diff
@@ -0,0 +1,7 @@
+[general]
+    tags = ["FixIgnoreLines"]
+    description = "Test for ignoreLines option in Fix"
+    suiteName = "Autofix: Smoke Tests"
+
+[fix]
+    ignoreLines = ["// IGNORE_ME"]
```

Comment:
may be test with regex here?

Reply:
and better to test with several different regexes

Reply:
Done